### PR TITLE
[#461] Improve bundle CSV

### PR DESCRIPTION
**Pull request by brusdev**
Add spec descriptors and improve CRD descriptions.

### CODE REVIEW

File: deploy/install/020_crd_artemis_security.yaml [hunk 1]
```diff
@@ -651,44 +669,48 @@ spec:
...
+                            description: Specify the role accesses
                             items:
                               properties:
                                 accessList:
+                                  description: Specify the default accesses
                                   items:
                                     properties:
                                       method:
-                                        description: The method of defaultAccess/roleAccess List
+                                        description: The method of the access entry
```
Comment (pwright) [inline deploy/install/020_crd_artemis_security.yaml:694]:
```suggestion
                                        description: Specifies the access entry method
```

File: deploy/install/020_crd_artemis_security.yaml [hunk 2]
```diff
@@ -651,44 +669,48 @@ spec:
...
                                         type: string
                                       roles:
-                                        description: The roles of defaultAccess/roleAccess List
+                                        description: The roles of the access entry
```
Comment (pwright) [inline deploy/install/020_crd_artemis_security.yaml:697]:
```suggestion
                                        description: Specifies the access entry roles
```

File: deploy/install/020_crd_artemis_security.yaml [hunk 3]
```diff
@@ -651,44 +669,48 @@ spec:
...
                                   type: string
                                 key:
-                                  description: The key of roleAccess List
+                                  description: The key of the role access
                                   type: string
                               type: object
                             type: array
                         type: object
                       connector:
+                        description: Specify the connector configurations
```
Comment (pwright) [inline deploy/install/020_crd_artemis_security.yaml:713]:
```suggestion
                        description: Specify connector configurations
```

**Review by pwright — COMMENTED** [review]
@brusdev  some suggestions

**Comment by brusdev** [comment]
@pwright thanks for you review, I moved your changes to the source files:
https://github.com/artemiscloud/activemq-artemis-operator/blob/main/api/v1beta1/activemqartemis_types.go
https://github.com/artemiscloud/activemq-artemis-operator/blob/main/api/v1beta1/activemqartemisaddress_types.go
https://github.com/artemiscloud/activemq-artemis-operator/blob/main/api/v1beta1/activemqartemisscaledown_types.go
https://github.com/artemiscloud/activemq-artemis-operator/blob/main/api/v1beta1/activemqartemissecurity_types.go

**Pull request merged**